ColabTool/Shared/PostHeader/index.jsx

```diff
@@ -6,12 +6,20 @@ import {
 import style from './style.css';
 
 const getPostDetailString = (draftDetails) => {
-  if (draftDetails.userName || draftDetails.email) {
-    if (draftDetails.via === 'api') {
-      return `${draftDetails.userName || draftDetails.email} created this ${draftDetails.createdAt}`;
+  const hasIdInfo = draftDetails.userName || draftDetails.email;
+
+  if (hasIdInfo) {
+    if (draftDetails.isRetweet) {
+      return `${hasIdInfo} created this retweet ${draftDetails.createdAt}`;
```

Comment:
I think it might be worth renaming `hasIdInfo` to just `idInfo` here since we're using it in the string and not just as a truthy/falsy thing 😄